lib/watirspec/server.rb

```diff
@@ -54,14 +54,14 @@ def run!
 
             status, headers, body = app.response(path, data)
 
-            headers = headers.merge('Server' => 'ruby')
+            headers['Server'] = 'ruby'
             headers = headers.to_a.map { |array| array.join(': ') }
 
             response = ["HTTP/1.1 #{status}"] + headers
             response = response.join("\r\n")
 
             client.puts(response)
-            client.puts("\r\n")
+            client.puts("\r\n\r\n")
```

Comment:
WAT. I'll take a look but that's actually the correct way ([see spec](https://www.w3.org/Protocols/rfc2616/rfc2616-sec6.html)) - there should be one CRLF at the end of header block and one CRLF before body.